packages/ui-kit/src/helpers/graphql/stories/MetricInput.stories.tsx

```diff
@@ -0,0 +1,45 @@
+import type { Meta, StoryObj } from '@storybook/react'
+import type { MetricInput } from '../index'
+
+export const MetricInputComponent: React.FC<MetricInput> = () => null
+
+const meta = {
+  title: 'API/MetricInput',
+  tags: ['pattern'],
+  component: MetricInputComponent,
+  argTypes: {
+    average: {
+      description: 'An ad hoc Average Metric.'
+    },
+    count: {
+      description: 'An ad hoc Count Metric.'
+    },
+    countDistinct: {
+      description: 'An ad hoc Count Distinct Metric.'
+    },
+    custom: {
+      description: 'An ad hoc Custom Metric.'
+    },
+    id: {
+      description: 'The ID of a pre-configured Metric.'
```

Comment:
We need to double-check the names of everything. Are these "ad hoc" and "pre-configured" metrics? Are these "inline" and "predefined" metrics? I'm not sure… cc @acossta